app/helpers/Emails/Notifications/Reviews/commentUpdated_en.latte

```diff
@@ -9,16 +9,17 @@
     that was closed by the teacher at <strong>{$closed|localizedDate:'en'}</strong>.
 </p>
 
-<h5>Updated at <code>{$comment->getFile()}:{$comment->getLine()}</code></h5>
+<h5 n:if="$comment->getFile()">Updated at <code>{$comment->getFile()}:{$comment->getLine()}</code></h5>
+<h5 n:if="!$comment->getFile()">Review summary updated</h5>
 
 {if $oldText !== $comment->getText()}
 <h6>Previous version:</h6>
-{$oldText|markdown}
+{$oldText|markdown|noescape}
```

Comment:
Do you really want to allow not escaped comments in emails? Isn't this a road to stored XSS?

Reply:
Ah, I see